### ISSUE TITLE
`mri_nu_correct.mni` errors out when trying to output help message

### ISSUE THREAD

**Issue by pwighton** [freesurfer/freesurfer#779]
### Platform details:

- FreeSurfer versions: 7.1 and 7.0
- Operating system name and version: The docker container `pwighton:freesurfer:7.1.0` built using [neurodocker](https://github.com/ReproNim/neurodocker/issues/333) and `continuumio/miniconda:4.7.12` as it's base

### Summary

When `mri_nu_correct.mni` tries to print it's help text via either `mri_nu_correct.mni` or `mri_nu_correct.mni --help`, the following error occurs:

```
I/O warning : failed to load external entity "/opt/freesurfer-7.1.0/docs/xml/mri_nu_correct.mni.help.xml"
Document not parsed successfully.
```

### Expected behavior
Should output the [help text](https://github.com/freesurfer/freesurfer/blob/fs-7.1/scripts/mri_nu_correct.mni.help.xml)

### How to replicate the behavior

```
docker run --rm -it \
  -v /tmp/fssubs:/fsdata \
  -v ${HOME}/license.txt:/license.txt \
  -e FS_LICENSE='/license.txt' \
  -e SUBJECTS_DIR='/fsdata' \
  --name fs7 \
  pwighton/freesurfer:7.1.0 \
    mri_nu_correct.mni
```
or
```
docker run --rm -it \
  -v /home/paul/lcn/data/fs-course-20200310/subjects:/fsdata \
  -v ${HOME}/license.txt:/license.txt \
  -e FS_LICENSE='/license.txt' \
  -e SUBJECTS_DIR='/fsdata' \
  --name fs7 \
  pwighton/freesurfer:7.1.0 \
    mri_nu_correct.mni --help
```

When the command is invoked and does not have to print the help message, it works fine:
```
mri_nu_correct.mni --i orig.mgz --o ~/delete_me_nu.mgz --n 2 --ants-n4
```

### Details

See [this conversation](https://github.com/ReproNim/neurodocker/issues/333#issuecomment-627840036)

**Comment by ahoopes** [comment]
Thanks Paul. This must have been missed in the cmake conversion. Fixed now with d5d0d76dcfc35fcf7d7a5abe929926e81bc8ccce in dev, which will make it into the next 7.1 patch

**Comment by pwighton** [comment]
Thanks @ahoopes!
